macros/dmt_get_test_sql.sql

```diff
@@ -123,13 +123,13 @@
     {% do run_query(get_merge_sql(model, test_sql, dest_columns=dest_columns)) %}
 {% endmacro %}
 
-{% macro __set_rendered_keys(ns, keys) %}  
-    {% for k in keys %}
-        {% do ns.rendered_keys.update({k: render("{{ " + k + " }}")}) %}
+{% macro __set_rendered_mappings(ns, input_mapping) %}

```

Comment:
FYI: This forces the ref in the `sql` files and the `yml` files to be EXACTLY the same